src/v2/Components/UserSettings/UserInformation/index.tsx

```diff
@@ -0,0 +1,220 @@
+  }
+
+  return (
+    <Box>
+      <Serif size="6" mb={space(2)}>
+        Information
+      </Serif>
+      <Formik
+        initialValues={me}
+        validationSchema={ChangeEmailValidator}
+        onSubmit={onSubmit}
+      >
+        {({
+          errors,
+          handleBlur,
+          handleChange,
+          handleSubmit,
+          isSubmitting,
+          status,
+          touched,
+          values,
+        }) => (
+          <Form onSubmit={handleSubmit}>
+            <Text>Full name</Text>
+            <QuickInput
+              error={errors.name as any}
+              placeholder="Enter your full name"
+              name="name"
+              type="text"
+              value={values.name}
+              onChange={handleChange}
+              onBlur={handleBlur}
+            />
+            <Text>Email</Text>
+            <QuickInput
+              block
+              error={errors.email as any}
+              placeholder="Enter your email address"
+              name="email"
+              type="email"
+              value={values.email}
+              onChange={handleChange}
+              onBlur={handleBlur}
+            />
+            <Text>Mobile number</Text>
+            <QuickInput
+              block
+              placeholder="Enter your mobile phone number"
+              name="phone"
+              type="tel"
+              value={values.phone}
+              onChange={handleChange}
+              onBlur={handleBlur}
+            />
+            {me.paddleNumber && (
+              <>
+                <Text>Bidder number</Text>
+                <QuickInput
+                  block
+                  name="paddleNumber"
+                  value={me.paddleNumber}
+                  readOnly
+                />
+              </>
+            )}
+            <input name="internalID" value={me.internalID} hidden readOnly />
+            {touched.email && values.email !== me.email && (
+              <>
+                <Text>Password</Text>
+                <PasswordInput
+                  autoFocus
+                  block
+                  error={
+                    !values.password && "Password is required to change email."
+                  }
+                  placeholder="Enter your password"
+                  name="password"
+                  value={values.password}
+                  onChange={handleChange}
+                  onBlur={handleBlur}
+                  required
+                />
+              </>
+            )}
+            {status && !status.success && <Error show>{status.error}</Error>}
+            <Button
+              type="submit"
+              size="large"
+              loading={isSubmitting}
+              width="100%"
+              variant="secondaryOutline"
+            >
+              Save changes
+            </Button>
+          </Form>
+        )}
+      </Formik>
+    </Box>
+  )
+}
+
+export const UserInformationRefetchContainer = createRefetchContainer(
```

Comment:
Possibly used the `TwoFactorAuthenticationRefetchContainer` component as a reference. We use a refetch container there in order to refetch enabled factors after enabling/disabling a factor. Agreed that this could probably be a fragment container instead.

https://github.com/artsy/force/blob/master/src/v2/Components/UserSettings/TwoFactorAuthentication/index.tsx#L50